src/pages/gl-account-groups/index.js

```diff
@@ -88,22 +104,10 @@ const AccountGroups = () => {
           pageSize={50}
           paginationType='client'
```

Comment:
pagination type is client ? if yes we dont need the return _startAt else we need 'api'